client/src/test/java/com/paypal/selion/testcomponents/BasicPageImplTest.java

```diff
@@ -57,16 +58,26 @@ public void testLoadObjectMapFromMap() throws InterruptedException, IOException
 
     @Test(groups = { "functional" })
     @WebTest
-    public void testWebPageIsValidated() throws InterruptedException, IOException {
+    public void testWaitUntilPageIsValidated() throws InterruptedException, IOException {
         page = new TestPage();
         Grid.open("about:blank");
         String script = getScript();
         Grid.driver().executeScript(script);
         Thread.sleep(4000);
 
-        WebDriverWaitUtils.waitUntilWebPageIsValidated(page);
-        //Note: if the previous line passes and this fails something is really broken
-        SeLionAsserts.assertTrue(page.isPageValidated(), "TestPage did not pass pageValidations");
+        WebDriverWaitUtils.waitUntilPageIsValidated(page);
+    }
+
+    @Test(groups = { "functional" }, expectedExceptions = { TimeoutException.class })
+    @WebTest
+    public void testWaitUntilPageIsValidated_Neg() throws InterruptedException, IOException {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/avnguyen/SeLion/file/1215908312/issues/source?bid=2928299&fileBranchId=2929577#l73)